packages/fx-core/src/component/generator/utils.ts

```diff
@@ -36,16 +36,17 @@ const sampleTagPrefix = sampleConfig.tagPrefix;
 const sampleTagListURL = sampleConfig.tagListURL;
 
 function selectTemplateTag(tags: string[]): string | undefined {
-  if (preRelease === "alpha") {
-    return templateAlphaVersion;
-  }
-  if (preRelease === "beta") {
-    return templateBetaVersion;
-  }
-  const versionPattern = preRelease ? `0.0.0-${preRelease}` : templateVersion;
-  const versionList = tags.map((tag: string) => tag.replace(templateTagPrefix, ""));
-  const selectedVersion = semver.maxSatisfying(versionList, versionPattern);
-  return selectedVersion ? templateTagPrefix + selectedVersion : undefined;
+  return templateAlphaVersion;
+  // if (preRelease === "alpha") {
```

Comment:
why not delete directly?

Reply:
These code are commented out temporarily. When v3 is ready they will be uncommented.

Reply:
This method is only called by v3. For v2 it still uses old ways to match tags.